### PR TITLE
Mvp_fixes

### PR DESCRIPTION
**Pull request by zalmane**

### CODE REVIEW

File: datayoga-py/datayoga/jmespath_custom_functions.py [hunk 1]
```diff
@@ -102,9 +103,10 @@ def _func_time_delta_days(self, dt):
         If `dt` is a number, Unix timestamp (1320365123, for example) is assumed.
         """
 
-        dt = datetime.fromisoformat(dt) if isinstance(dt, str) else datetime.fromtimestamp(dt)
+        dt = datetime.fromisoformat(dt) if isinstance(dt, str) else datetime.fromtimestamp(dt, timezone.utc)
+        print(dt.now(dt.tzinfo), datetime.now(dt.tzinfo), dt)
         delta = dt.now(dt.tzinfo) - dt
-
+        print(delta)
```
Comment (spicy-sauce) [inline datayoga-py/datayoga/jmespath_custom_functions.py]: debug message, please remove.

File: datayoga-py/datayoga/jmespath_custom_functions.py [hunk 2]
```diff
@@ -102,9 +103,10 @@ def _func_time_delta_days(self, dt):
         If `dt` is a number, Unix timestamp (1320365123, for example) is assumed.
         """
 
-        dt = datetime.fromisoformat(dt) if isinstance(dt, str) else datetime.fromtimestamp(dt)
+        dt = datetime.fromisoformat(dt) if isinstance(dt, str) else datetime.fromtimestamp(dt, timezone.utc)
+        print(dt.now(dt.tzinfo), datetime.now(dt.tzinfo), dt)
```
Comment (spicy-sauce) [inline datayoga-py/datayoga/jmespath_custom_functions.py]: debug message, please remove.

File: datayoga-py/pyproject.toml [hunk 3]
```diff
@@ -38,7 +38,7 @@ SQLAlchemy = { version = "^1.4.41", optional = true}
 [tool.poetry.extras]
 redis = ["redis"]
 pg = ["psycopg2","SQLAlchemy"]
-test = ["mock", "pytest", "pytest-describe", "pytest-mock", "pytest-timeout", "requests-mock", "redis", "psycopg2","SQLAlchemy", "testcontainers"]
+test = ["mock", "pytest", "pytest-describe", "pytest-mock", "pytest-timeout", "requests-mock", "redis", "psycopg2","SQLAlchemy", "testcontainers", "pytest-asyncio"]
```
Comment (spicy-sauce) [inline datayoga-py/pyproject.toml]: please list after "pytest" and before "pytest-describe" in this list.

**Review by spicy-sauce — CHANGES_REQUESTED** [review]

File: datayoga-py/datayoga/jmespath_custom_functions.py [hunk 4]
```diff
@@ -86,7 +86,8 @@ def prepare() -> Union[bytes, bytearray]:
             if isinstance(obj, str):
                 return obj.encode()
 
-            return orjson.dumps(obj)
+            # the 'separators' arg is needed to remove whitespace in the resulting string
+            return json.dumps(obj, separators=(',', ':')).encode("utf-8")
```
Comment (spicy-sauce) [inline datayoga-py/datayoga/jmespath_custom_functions.py:90]: probably we should have such a function in utils.

**Review by spicy-sauce — CHANGES_REQUESTED** [review]

File: datayoga-py/datayoga/jmespath_custom_functions.py (hunk 1 above)
Comment (zalmane) [reply datayoga-py/datayoga/jmespath_custom_functions.py]: removed

File: datayoga-py/datayoga/jmespath_custom_functions.py (hunk 2 above)
Comment (zalmane) [reply datayoga-py/datayoga/jmespath_custom_functions.py]: removed

File: datayoga-py/pyproject.toml (hunk 3 above)
Comment (zalmane) [reply datayoga-py/pyproject.toml]: done

File: datayoga-py/datayoga/jmespath_custom_functions.py (hunk 4 above)
Comment (zalmane) [reply datayoga-py/datayoga/jmespath_custom_functions.py:90]: left as is

**Review by spicy-sauce — APPROVED** [review]

**Pull request merged**